Tests/src/PublicAPITests/SpaceSystemTests.cpp

```diff
@@ -98,6 +98,18 @@ void CreateSpaceWithBuffer(::SpaceSystem* SpaceSystem,
 	OutSpace = Result.GetSpace();
 }
 
+void CreateDefaultTestSpace(SpaceSystem* SpaceSystem, Space& OutSpace)
+{
+	// Create space
+	const char* TestSpaceName		 = "OLY-UNITTEST-SPACE-MAG";
+	const char* TestSpaceDescription = "OLY-UNITTEST-SPACEDESC-MAG";
+
+	char UniqueSpaceName[256];
+	SPRINTF(UniqueSpaceName, "%s-%s", TestSpaceName, GetUniqueString().c_str());
+
+	CreateSpace(SpaceSystem, UniqueSpaceName, TestSpaceDescription, SpaceAttributes::Private, nullptr, nullptr, nullptr, OutSpace);
+}
```

Comment:
[Consider][Strong] I don't see why you wouldn't create the space object in this method and return it, rather than use a `out` param.
You could even move the system setup and whatnot into here, shorten all the above tests.